From 6f5b26e5ce034336154f91ab8c3e64bdb43062a0 Mon Sep 17 00:00:00 2001
From: Jaegeuk Kim <jaegeuk@google.com>
Date: Wed, 1 Apr 2026 18:15:37 +0000
Subject: [PATCH 52/63] ANDROID: Update symbol list for Pixel

Summary:

1 function symbol(s) added
  'int __traceiter_android_vh_ra_alloc_retry(void*, unsigned int*, bool*)'

1 variable symbol(s) added
  'struct tracepoint __tracepoint_android_vh_ra_alloc_retry'

Bug: 498623919
Change-Id: I9aab93d0beaa5ca159bd065f8dd657a24ea185a9
Signed-off-by: Jaegeuk Kim <jaegeuk@google.com>
---
 gki/aarch64/abi.stg       | 27 +++++++++++++++++++++++++++
 gki/aarch64/symbols/pixel |  4 ++++
 2 files changed, 31 insertions(+)

diff --git a/gki/aarch64/abi.stg b/gki/aarch64/abi.stg
index 4a3e4da2a160..93285a5b5495 100644
--- a/gki/aarch64/abi.stg
+++ b/gki/aarch64/abi.stg
@@ -419656,6 +419656,13 @@ function {
   parameter_id: 0x1b8590a8
   parameter_id: 0x084ef4c8
 }
+function {
+  id: 0x9bc40c18
+  return_type_id: 0x6720d32f
+  parameter_id: 0x18bd6530
+  parameter_id: 0x1bf16028
+  parameter_id: 0x11cfee5a
+}
 function {
   id: 0x9bc482f2
   return_type_id: 0x6720d32f
@@ -444551,6 +444558,15 @@ elf_symbol {
   type_id: 0x9b9f29ea
   full_name: "__traceiter_android_vh_queue_io"
 }
+elf_symbol {
+  id: 0x86490c74
+  name: "__traceiter_android_vh_ra_alloc_retry"
+  is_defined: true
+  symbol_type: FUNCTION
+  crc: 0xb51f5a22
+  type_id: 0x9bc40c18
+  full_name: "__traceiter_android_vh_ra_alloc_retry"
+}
 elf_symbol {
   id: 0x3a545b61
   name: "__traceiter_android_vh_ra_tuning_max_page"
@@ -452390,6 +452406,15 @@ elf_symbol {
   type_id: 0x18ccbd2c
   full_name: "__tracepoint_android_vh_queue_io"
 }
+elf_symbol {
+  id: 0x972d060a
+  name: "__tracepoint_android_vh_ra_alloc_retry"
+  is_defined: true
+  symbol_type: OBJECT
+  crc: 0x7c5aa8a7
+  type_id: 0x18ccbd2c
+  full_name: "__tracepoint_android_vh_ra_alloc_retry"
+}
 elf_symbol {
   id: 0x811d5fab
   name: "__tracepoint_android_vh_ra_tuning_max_page"
@@ -527392,6 +527417,7 @@ interface {
   symbol_id: 0x358b7cff
   symbol_id: 0x1adac14e
   symbol_id: 0x354db2e1
+  symbol_id: 0x86490c74
   symbol_id: 0x3a545b61
   symbol_id: 0xaa42bd9c
   symbol_id: 0xcf2bdce0
@@ -528263,6 +528289,7 @@ interface {
   symbol_id: 0x428834b1
   symbol_id: 0x74120954
   symbol_id: 0x3ac6daa7
+  symbol_id: 0x972d060a
   symbol_id: 0x811d5fab
   symbol_id: 0xf0a69156
   symbol_id: 0x9cdcdb66
diff --git a/gki/aarch64/symbols/pixel b/gki/aarch64/symbols/pixel
index ccf9b04e589d..ef67a9eb833f 100644
--- a/gki/aarch64/symbols/pixel
+++ b/gki/aarch64/symbols/pixel
@@ -61,6 +61,7 @@
   bio_endio
   bio_end_io_acct_remapped
   bio_init
+  bio_put
   bio_start_io_acct
   bitmap_alloc
   __bitmap_andnot
@@ -344,6 +345,7 @@
   __dev_change_net_namespace
   dev_close
   dev_coredumpm_timeout
+  dev_coredumpv
   _dev_crit
   dev_driver_string
   _dev_emerg
@@ -2840,6 +2842,7 @@
   __traceiter_android_vh_prio_inheritance
   __traceiter_android_vh_prio_restore
   __traceiter_android_vh_put_task
+  __traceiter_android_vh_ra_alloc_retry
   __traceiter_android_vh_resume_end
   __traceiter_android_vh_rmqueue
   __traceiter_android_vh_scheduler_tick
@@ -2995,6 +2998,7 @@
   __tracepoint_android_vh_prio_inheritance
   __tracepoint_android_vh_prio_restore
   __tracepoint_android_vh_put_task
+  __tracepoint_android_vh_ra_alloc_retry
   __tracepoint_android_vh_resume_end
   __tracepoint_android_vh_rmqueue
   __tracepoint_android_vh_scheduler_tick
-- 
2.50.1

